resources/md/guestbook.md

```diff
@@ -464,7 +464,10 @@ The key references the `:db.sql/query-fn` component which is responsible for ins
  :filename "sql/queries.sql"}
 ```
 
-Now that our model is all set up, let's start the application.
+Like in the REPL example towards the end of the [Accessing the Database](accessing_the_database) section, the `:db.sql/query-fn` component comes from kit-sql. Unlike in that example:
```

Comment:
Like with that REPL example, I'm not sure this part is correct. Would appreciate feedback.

Reply:
I think it might make more sense to move this down to `Creating a controller for the guestbook` section below since that's where we're going to deal with making a controller and accessing the db query functions. We could mention that when the application starts up the connection for the database will be loaded based on `resources/system.edn` configuration.

Reply:
Do you mean simply moving lines 469-474 directly to 510 (end of the controller section)?
I thought that in the controller section we are dealing with this a bit less explicitly  I'm not sure lines 471 and 472 make sense in there?

Reply:
Ah ok, I looked closer and we can keep it as is. I thought we still needed Running the Application section, but we already do a restart after adding sqlite module in and that section isn't needed after all.